tests/rados/test_pg_split.py

```diff
@@ -0,0 +1,87 @@
+import traceback
```

Comment:
Please add docstrings for the module, explaining the intent.

Reply:
done